web/client-ide/nginx/init-notebooks.sh

```diff
@@ -0,0 +1,5 @@
+set -eux
```

Comment:
Add `#!/usr/bin/env sh`.

I prefer to be explicit in scripts:

```sh
set -o errexit
set -o pipefail
set -o nounset
```

I don't think we need `x` / `xtrace`.